compiler/luci/pass/src/VerifyQuantizedNodeChannelWiseGranularity.h

```diff
@@ -32,9 +32,30 @@ using Granularity = luci::QuantizationGranularity;
 namespace luci
 {
 
+/**
+ * @brief Verify the granualrity of channel-wise quantized node
+ * @details
+ *
+ * Targets to verify
+ * - node's output (i.e., node itself)
+ * - node's inputs
+ */
 struct VerifyQuantizedNodeChannelWiseGranularity final : public luci::CircleNodeVisitor<bool>
 {
 private:
+  bool is_lwq(const loco::Node *node)
```

Comment:
We are dealing only with CircleNode in luci. Is there any particular reason to accept loco::Node ?

Reply:
That's because it is more concise.

For example, the current code looks as below.

```cpp
  bool visit(const luci::CircleMul *node)
  {
    RETURN_FALSE_UNLESS(is_lwq(node))
    RETURN_FALSE_UNLESS(is_lwq(node->x()));
    RETURN_FALSE_UNLESS(is_lwq(node->y()));
    return true;
  }
```

If we use `CircleNode` as the argument type, type casting should be inserted to all places where the accessor of CircleNode is called. Like,
 
`RETURN_FALSE_UNLESS(is_lwq(loco::must_cast<const luci::CircleNode>(node->x())));`